src/lib/styles.tsx

```diff
@@ -0,0 +1,49 @@
+export const stylify = async (element: HTMLElement): Promise<void> => {
```

Comment:
Minor: As of now at least, I don't think we need to make this `async` (it doesn't have any `await`s or anything). That would allow you to remove some of the `.catch()` when invoking.